ts/defs.bzl

```diff
@@ -172,6 +173,12 @@ def ts_project(
 
             See [docs/transpiler.md](/docs/transpiler.md) for more details.
 
+        declaration_transpiler: A custom transpiler tool to run that produces the TypeScript declaration outputs instead of `tsc`.
+
+            This is only used when `transpiler` is set to a custom transpiler tool.
```

Comment:
Is this true? I don't see it asserted. A user could use tsc for transpilation but configure it to not emit declarations and use a custom declaration_transpiler right?

Reply:
I think you're correct. This is not true, I originally thought it would be but I think this full PR turned out a bit simpler then I originally thought 👍